### PR TITLE
Add nrlw icon

### PR DESCRIPTION
**Pull request by brunano21**
nrwl

**Issue:** Contributes to #4767
**Alexa Rank:** [350,891](https://www.alexa.com/siteinfo/nrwl.io)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5005,6 +5005,11 @@
             "hex": "CB3837",
             "source": "https://github.com/npm/logos"
         },
+        {
+            "title": "Nrwl",
+            "hex": "96D7E8",
+            "source": "https://nrwl.io/assets/nrwl-logo-white.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:5011]: Could you please update the source to the homepage? The SVG itself is embedded in the header, and will likely be kept that way even if the file path changes in the future.

File: icons/nrwl.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....17 4.43 4.43 0 002.989-.193.528.528 0 00.115-.076.179.179 0 00.076.03l1.789.169v.863H8.75v-.734l1.12.105-4.204-.754a.111.111 0 00-.014-.004c-.01-.1-.095-.172-.13-.218a.134.134 0 01-.03-.07 1.64 1.64 0 00-1.496-1.52.504.504 0 00-.18-.193.503.503 0 01-.187-.4.56.56 0 00-.498.44 1.393 1.393 0 01-.377-2.222 1.798 1.798 0 011.312-.563A1.824 1.824 0 015.83 10.96a.914.914 0 01-.543.32.904.904 0 00-.736.748c.35 0 .585.477 1.236.477a.59.59 0 00.547-.367.592.592 0 00.549.367 1.17 1.17 0 00.49-.106v2.002l1.377.327v-1.592l2.193 2.605H12.1v-4.89h-1.38v2.605L8.53 10.852H7.373v.427c-.283-.05-.556-.255-.65-.52a3.424 3.424 0 00-3.366-2.501zM22.109 9.96v4.414c0 1.186.766 1.336 1.485 1.336.219 0 .406-.02.406-.02v-1.03s-.078.007-.164.007c-.305 0-.375-.12-.375-.453V9.96zm-6.816 1.932a2.057 2.057 0 00-.709.128 1.253 1.253 0 00-.535.385v-.453h-1.285v3.79h1.347v-1.74c0-.316.081-.551.244-.704.164-.154.39-.23.678-.23a2.937 2.937 0 01.307.017v-1.193a2.057 2.057 0 00-.047 0zm.584.06l1.094 3.787h1.306l.477-1.736.45 1.736h1.306l1.094-3.787h-1.186l-.54 1.932-.52-1.932h-1.15l-.542 1.912-.512-1.912zm-12.281 2.14c.03 0 .07.016.117.027.092.023.17.02.15.108a.171.171 0 01-.332-.08c.01-.044.033-.056.065-.055z"/></svg>
```
Comment (adamrusted) [inline icons/nrwl.svg:1]: This icon looks spot on, though I think we can get away with just the narwhal (without the text) as they do on their social media channels.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks @brunano21 - some great work here! 🎉
I've made a couple of comments below, if you could look into those that'd be great! 💪🏻

File: icons/nrwl.svg (hunk 2 above)
Comment (brunano21) [reply icons/nrwl.svg:1]: Ok - this requires me to work again on the icon. Stay tuned!

File: _data/simple-icons.json (hunk 1 above)
Comment (brunano21) [reply _data/simple-icons.json:5011]: Sure will do!

File: icons/nrwl.svg (hunk 2 above)
Comment (brunano21) [reply icons/nrwl.svg:1]: Now that I think more I believe I wondered the same - initially I thought to remove the whole text and leave the narwhal only. However, when I was going to do that I realized that a piece of the tail was missing and I don't know I could recreate that :-/
What do you think?

Comment (adamrusted) [reply icons/nrwl.svg:1]: Hmm...

@kmaida are you able to help at all? We're wanting to use just the narwhal as seen in the social icons (and favicon) of [nrwl.io](https://nrwl.io/). I could probably look to rebuild the tail, but if there is an official file that contains it - saves me doing any guess-work! 😅

Comment (adamrusted) [reply icons/nrwl.svg:1]: @brunano21 - I'll look to merge this for now, being as your icon is spot on. If @kmaida or another maintainer for nrwl could advise on whether there is a version without the tail cut off, we can look to update the icon at a later date.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @brunano21 🐬

**Pull request merged**